spec/ace.md

```diff
@@ -132,41 +132,62 @@ An executor MAY implement a "strict mode" where an image cannot run unless all i
 These isolators are specific to the Linux kernel and are impossible to represent as a 1-to-1 mapping on other kernels.
 The first example is "capabilities" but this will be expanded to include things such as SELinux, SMACK or AppArmor.
 
-#### os/linux/capabilities-remove-set
+#### os/linux/capabilities-retain-set
 
 * Scope: app
 
 **Parameters:**
 
-* **set** list of capabilities that will be removed from the process's capabilities bounding set, all others will be included.
+* **set** list of capabilities that will be added in the process's capabilities bounding set in addition to the default app capabilities defined thereafter.
```

Comment:
Well, I'd been considering them mutually exclusive. Either you use the retain-set in which case you're specifying exactly which capabilities you want, or you specify nothing in which case you get the default set, or you specify remove-set in which case you get the default set minus remove-set.